packages/admin-ui/src/combobox/combobox.state.ts

```diff
@@ -34,8 +34,7 @@ export function useComboboxState<T>(
   const [selectedItem, setSelectedItem] = useState<T>()
 
   // data states
-  const [loading, setLoading] = useState(false)
-  const [error, setError] = useState(false)
+  const [userStatus, setStatus] = useState<ComboboxStatus>()
```

Comment:
```suggestion
  const [internalStatus, setStatus] = useState<ComboboxStatus>()
```
maybe?

Reply:
Isn't it more like an external status since the user set it?